### DISCUSSION TITLE
Server down for the last 2 hours

### DISCUSSION THREAD

**Discussion by jakerieger** [planetscale/discussion#222] [Bug Reports] [answered]
Our team is unable to connect to the database instance. Dropped connection at around 4:45PM EST and has been down since. Appears to be the AWS instance itself that's down. Don't see an option to restart the instance or anything so posting this here.

**Comment by jakerieger** [comment, marked as answer]
Seems like it just came back online. I'm curious if there's a plan to provide more options for troubleshooting downtime? We've had this database up for less than 48 hours now and although PS seems very promising, this incident hasn't been very confidence-inspiring.

**Reply by jgreet** [reply]
Hi @jakerieger, sorry for the interruption. Is your database in US-East? There was an issue that caused intermittent connection issues in that region today.

Status updates were posted here: https://www.planetscalestatus.com/incidents/2gr0nsswn2pw

**Reply by jakerieger** [reply]
It was, Virginia. Good to know, appreciate the response

**Reply by chuanhhoang** [reply]
2 hour down time is too much. What is the SLA?

**Reply by isaac-mcfadyen** [reply]
> 2 hour down time is too much. What is the SLA?

Planetscale's SLA is [here](https://planetscale.com/legal/planetscale-service-level-agreement). Sounds like that's for enterprise customers with a contract though (not sure what the contract level was here).

**Reply by jakerieger** [reply]
At the moment, we're just on the hobby tier testing things out. I agree that 2 hours is too long of a period for uninterrupted downtime but I also can't expect too much from a free tier. My biggest concern was a lack of ability to investigate through the PL platform why our connection was dropped. Some tools on the dashboard for server health would be a bare minimum feature imo and I hope they implement something down the road

**Reply by johnyerhot** [reply]
> My biggest concern was a lack of ability to investigate through the PL platform why our connection was dropped. Some tools on the dashboard for server health would be a bare minimum feature imo and I hope they implement something down the road

We hear that. Part of what's difficult with this one in particular is that the cause of the connection issues was not located with the database clusters themselves but rather in the service that manages external connection sessions.It's unfortunate but also produced work to make that part of PlanetScale more resilient. How outage information is displayed to y'all, beyond a status page, is also something we've been chatting about.

> Planetscale's SLA is [here](https://planetscale.com/legal/planetscale-service-level-agreement). Sounds like that's for enterprise customers with a contract though (not sure what the contract level was here).

That is accurate.
